Tweeter/Scripts/app.js

```diff
@@ -0,0 +1,29 @@
+﻿var app = angular.module('Tweeter', []);
+
+app.controller('Register', function ($scope, $q, $http) {
+
+    $scope.userNameExistsCode = null;

```

Comment:
@katerebekah
This is simply to use a falsy value of `null` if the function hasn't run yet just for the ng-class.
ng-class:{'disabled': (!userNameExistsCode || userNameExistsCode == 1)}